src/linagora.esn.calendar/app/components/calendar-color-picker/calendar-color-picker.less

```diff
@@ -9,11 +9,44 @@
     margin: 3px;
     .all-centered;
   }
+  .square-box-color-picker{
+    .all-centered;
+    margin: 3px;
+    border: 2px dashed #B5B5B5;
+  }
   .color-box {
     width: 36px;
     height: 36px;
   }
   .selected {
     border: 1px solid @m-red;
   }
+  .calendar-theme-color-picker{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 39px;
+    height: 39px;
+  }
+  .calendar-theme-color-picker:hover{
+    cursor: pointer;
+  }
+
+  input[type="color"] {
+    -webkit-appearance: none;
+    border: none;
+  }
+  input[type="color"]::-webkit-color-swatch-wrapper {
+    padding: 0;
+  }
+  input[type="color"]::-webkit-color-swatch {
+    border: none;
+  }
```

Comment:
Will these pollute `input[type="color"]` everywhere? Shouldn't you put it in some local CSS scope?